cmake/macros/macro_deal_ii_package_handle.cmake

```diff
@@ -154,7 +154,10 @@ MACRO(DEAL_II_PACKAGE_HANDLE _feature _var)
     ENDFOREACH()
 
     #
-    # Remove certain system libraries from the link interface
+    # Remove certain system libraries from the link interface. This is
+    # purely cosmetic (we always implicitly link against the C library, and
+    # we always set up threading by linking against libpthread.so if
+    # necessary).
```

Comment:
@bangerth Updated. This is really a mainly cosmetic thing. Having "c" or all variants of "-pthread" "-lpthread" showing up in the link interface isn't pretty.